### ISSUE TITLE
androidx.test version resolution

### ISSUE THREAD

**Issue by rlatapy-luna** [jmfayard/refreshVersions#479]
- [x] I have read the [guidelines for submitting issues](https://jmfayard.github.io/refreshVersions/contributing/submitting-issues/)
## 🐛 Describe the bug
Both `AndroidX.test.runner` and `AndroidX.test.orchestrator` seems to use `version.androidx.test` but the version are not aligned.

## ⚠️ Current behavior
When I add these deps
```kotlin
    androidTestUtil(AndroidX.test.orchestrator)
    androidTestImplementation(AndroidX.test.runner)
```
it generates these versions
```
version.androidx.test=1.4.0
##        # available=1.4.1-alpha01
##        # available=1.4.1-alpha02
##        # available=1.4.1-alpha03
##        # available=1.4.1-beta01
##        # available=1.4.1-rc01
##        # available=1.4.1
```
But as of today, `orchestrator` last version is 1.4.1 and `runner` last version is 1.4.0
If I remove `orchestrator` dependency (or hardcode the version), I got these versions
```
version.androidx.test=1.4.0
##        # available=1.4.1-alpha01
##        # available=1.4.1-alpha02
##        # available=1.4.1-alpha03
```

## ✅ Expected behavior
Separated versions

## 💣 Steps to reproduce
- Add these deps
```kotlin
    androidTestUtil(AndroidX.test.orchestrator)
    androidTestImplementation(AndroidX.test.runner)
```
- Sync / refreshVersions

**Comment by LouisCAD** [comment]
What the AndroidX team is doing is a nightmare 🥲

I'm pretty sure it's a mistake on their end

**Comment by LouisCAD** [comment]
Fixed in 0.30.2-SNAPSHOT, I will publish a non snapshot release very soon, hopefully less than a day from now.

I reviewed this page: https://developer.android.com/jetpack/androidx/releases/test and it looks like they changed the versioning policy, or I inferred a wrong one. Anyway, you can see in the commit that closes this issue that the version keys will change (don't worry, refreshVersions will move to the new version keys, while keeping the version you had).

**Comment by LouisCAD** [comment]
Fix released in version 0.32.0
